features/networking/multicast.feature

```diff
@@ -273,3 +275,344 @@ Feature: testing multicast scenarios
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/networking/multicast-rc.json" replacing paths:
+      | ["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    Given 1 pods become ready with labels:
+      | name=mcast-pods |
+    And evaluation of `pod.ip` is stored in the :proj2_podip clipboard
+    And evaluation of `pod.name` is stored in the :proj2_pod clipboard
+
+    # enable multicast for the netnamespace
+    When I run the :annotate admin command with:
+      | resource     | netnamespace    |
+      | resourcename | <%= cb.proj2 %> |
+      | overwrite    | true            |
+      | keyval       | netnamespace.network.openshift.io/multicast-enabled=true |
+
+    # run omping on pod in first project
+    Given I use the "<%= cb.proj1 %>" project
+    When I run the :exec background client command with:
+      | pod              | <%= cb.proj1_pod %>   |
+      | oc_opts_end      |                       |
+      | exec_command     | omping                |
+      | exec_command_arg | -c                    |
+      | exec_command_arg | 5                     |
+      | exec_command_arg | -T                    |
+      | exec_command_arg | 15                    |
+      | exec_command_arg | <%= cb.proj1_podip %> |
+      | exec_command_arg | <%= cb.proj2_podip %> |
+    Then the step should succeed
+
+    # check the omping result on pod in second project
+    Given I use the "<%= cb.proj2 %>" project
+    When I run the :exec background client command with:
+      | pod              | <%= cb.proj2_pod %> |
+      | oc_opts_end      |                     |
+      | exec_command     | sh                  |
+      | exec_command_arg | -c                  |
+      | exec_command_arg | omping -c 5 -T 10 <%= cb.proj1_podip %> <%= cb.proj2_podip %> > /tmp/p2.log |
+    Then the step should succeed
+
+    When I execute on the "<%= cb.proj2_pod %>" pod:
+      | netstat | -ng |
+    Then the step should succeed
+    And the output should match:
+      | eth0\s+1\s+232.43.211.234 |
+
+    Given 10 seconds have passed
+    When I execute on the "<%= cb.proj2_pod %>" pod:
+      | cat | /tmp/p2.log |
+    Then the step should succeed
+    And the output should not contain:
+      | multicast, xmt/rcv/%loss = 5/5/0%" |
+
+  # @author hongli@redhat.com
+  # @case_id OCP-12966
+  @admin
+  @destructive
+  Scenario: pods in default project should be able to receive multicast traffic from other default project pods
+    Given the env is using multitenant or networkpolicy network
+    
+    # enable multicast and create testing pods
+    Given I switch to cluster admin pseudo user
+    And I use the "default" project
+    Given admin adds and overwrites following annotations to the "default" netnamespace:
+      | netnamespace.network.openshift.io/multicast-enabled=true |
+    Then the step should succeed
+
+    And I wait up to 120 seconds for the steps to pass:
```

Comment:
@weliang1, don't need to wrap around the step as there's a time-out within that step already.

Reply:
@pruan-rht Above test steps are new added and really needed. The reason is after combine all the multicast together and run them all, this case always fail. After debugging and find out the pods with name=mcast-pods are Terminated but not clean up, those pods are created in default namespace in previous case OCP-12931. For these two test cases, pods are created in default namespace, deleting the pods are take a while, without the steps to check if the pods are really clean up, this case always fail.